package.json

```diff
@@ -52,7 +52,8 @@
         "fast-memoize": "2.5.1",
         "prop-types": "^15.7.2",
         "rc-slider": "8.6.13",
-        "react-modal": "3.9.1"
+        "react-modal": "3.9.1",
+        "react-transition-group": "4.3.0"
```

Comment:
if we are planning to use transitions in JF and Search, maybe this should be a peer dependency so that we don't load it twice on the page.

Reply:
Make sense, thanks